### ISSUE TITLE
Write tests for all telemetry collection (that doesn't have it)

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5175]
Follow up from [https://mozilla-hub.atlassian.net/browse/VPN-3055](https://mozilla-hub.atlassian.net/browse/VPN-3055|smart-link). Was gonna write tests in there, but that is A LOT.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-3480)
┆Reporter: Beatriz Rizental Machado

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

I feel like we should actually go thorugh an auditing process for our all of our telemetry and once that is done we can write tests for each of the remaining collections. For now, this is a won't do. It's unnecessary extra work, even thought it would be a good exercise.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
